.github/workflows/deploy-unsigned.yml

```diff
@@ -81,9 +81,17 @@ jobs:
                        "ghcr.io/${{ github.repository }}/${IMG_NAME}:latest"
           done
 
+      - name: "Cancelled Deploy Images Job"
+        id: cancelled-job
+        if: cancelled()
+        run: echo "cancelled_job=true" >> $GITHUB_OUTPUT
+
```

Comment:
Could you remove the 2 `cancelled-job` steps if you added/moved it to the notify-slack job?
According to https://docs.github.com/en/actions/learn-github-actions/expressions#cancelled, the single `cancelled-job` step will trigger if the workflow is cancelled so you don't need the step in each job.